interfaces/cython/cantera/test/test_onedim.py

```diff
@@ -853,3 +853,40 @@ def solve(self, phi, T, width, P):
 
     def test_case1(self):
         self.solve(phi=0.4, T=300, width=0.05, P=0.1)
+
+
+class TestIonFlame(utilities.CanteraTest):
+    def test_ion_profile(self):
+        reactants = 'CH4:0.216, O2:2'
+        p = ct.one_atm
+        Tin = 300
+        width = 0.03
+
+        # IdealGasMix object used to compute mixture properties
+        self.gas = ct.Solution('ch4_ion.cti')
+        self.gas.TPX = Tin, p, reactants
+        self.sim = ct.IonFlame(self.gas, width=width)
+        self.sim.set_refine_criteria(ratio=4, slope=0.8, curve=1.0)
+
+        # stage one
+        self.sim.solve(loglevel=0, auto=True)
+        T1 = self.sim.T[-1]
+
+        # stage two
+        self.sim.solve(loglevel=0, stage=2, enable_energy=False)
+
+        # stage two
+        self.sim.solve(loglevel=0, stage=2, enable_energy=True)
+        Electron2 = self.sim.value(self.sim.flame, 'E', self.sim.flame.n_points-1)
+
+        #stage three
+        self.sim.solve(loglevel=0, stage=3, enable_energy=True)
+        Electron3 = self.sim.value(self.sim.flame, 'E', self.sim.flame.n_points-1)
+        T3 = self.sim.T[-1]
+
+        # check Temperature at outlet
+        self.assertNear(T1, T3, 1e-3)
+        self.assertNotEqual(T1, T3)
+        # check Electron concentration at outlet
+        self.assertNear(Electron2, Electron3, 1e-13)
+        self.assertNotEqual(Electron2, Electron3)
```

Comment:
I'm curious about the meaning of these assertions. 

What is the purpose of testing that the temperature changes between stages 1 and 3, but that the change is less than 0.1%?

Similarly, what is the purpose of testing that the electron mass fraction at the exit changes very little between stages 2 and 3, but is not exactly the same?

Are there any more useful tests that we can use? Even a rough regression test (i.e. that the peak E mass fraction is in some range) might be more useful.